docs/test/replication-controller-list.md

```diff
@@ -9,25 +9,19 @@ Cluster with few deployed applications, where:
 ## Expected result
 Page displays correctly, especially:
 
-* cards are displayed correctly even when page is scaled down/up
-* application details are valid and matching data from `kubectl` command
-* logs menu display correct pods information
-* all application actions are available
-* invalid applications have warning symbols and error messages
+* toolbar is displayed correctly
+* logs are displayed correctly
 
 ## Specific factors
 
-| Factor                               | 1                             | 2                                   | 3              | 4            | 5               | 6                         | 7          | 8      | Comment                                                                                     |
-|--------------------------------------|-------------------------------|-------------------------------------|----------------|--------------|-----------------|---------------------------|------------|--------|---------------------------------------------------------------------------------------------|
-| Replication controller count on page | 1                             | odd                                 | even           | 20-30        |                 |                           |            |        |                                                                                             |
-| Screen size                          | normal                        | mobile                              | narrow         | wide         | active resizing |                           |            |        |                                                                                             |
```

Comment:
For RC list page, the layouting is a very important feature. That's why we should test screen sizes in very detail. I don't see it for logs page. I would leave it to the general factor 'screen size'